src/commands/syncModel/run/README.md

```diff
@@ -1,101 +1,249 @@
+npx @kontent-ai/data-ops@latest sync-model run --help
+```
+
+## Sync Model Programmatically
+
+You can synchronize the content model within your scripts using the `syncModelRun` function:
+
+```typescript
 import { syncModelRun, SyncModelRunParams } from "@kontent-ai/data-ops";
 
 const params: SyncModelRunParams = {
-  sourceEnvironmentId: "<source-env-id>",
-  sourceApiKey: "<source-mapi-key>",
   targetEnvironmentId: "<target-env-id>",
-  targetApiKey: "<target-mapi-key>",
+  targetApiKey: "<target-api-key>",
+  // Use either sourceEnvironmentId and sourceApiKey, or folderName
+  // sourceEnvironmentId: "<source-env-id>",
+  // sourceApiKey: "<source-api-key>",
+  folderName: "./content-model",
   entities: {
-    contentTypes: () => true, // will sync
-    taxonomies: () => false, // won't sync
-    languages: (lang) => lang.codename === "default" // will sync only codename with default codename
+    contentTypes: () => true, // Synchronize all content types
+    taxonomies: (taxonomy) => taxonomy.codename.startsWith('category_'), // Synchronize specific taxonomies
+    languages: (language) => language.codename === "default" // Synchronize only the default language
   }
 };
 
 await syncModelRun(params);
 ```
 
+### Advanced Filtering
+
+When synchronizing programmatically, you can provide custom filter functions for each entity type:
+
+```typescript
+entities: {
+  contentTypes: (contentType) => contentType.codename !== 'obsolete_type',
+  taxonomies: (taxonomy) => taxonomy.terms.length > 0,
+  // Other entities...
+}
+```
+
+## Additional Notes
+
+- **Review Changes**: Always review the diff generated before synchronization to understand the changes that will be applied.
+- **Backup**: Consider exporting the target environment's content model before synchronization as a backup.
+- **Testing**: Test the synchronization in a non-production environment to ensure it behaves as expected.
+- **Dependencies**: Ensure that all dependencies (e.g., referenced taxonomies, content types) are included or exist in the target environment.
+
+## Common Use Cases
+
+- **Promoting Changes**: Move content model changes from Development to Production environments.
+- **Environment Refresh**: Keep Staging and Development environments in sync with Production.
+- **Automation**: Integrate content model synchronization into CI/CD pipelines.
+
+## Troubleshooting
+
+- **Validation Errors**: If synchronization fails due to validation errors, check the error messages and ensure all conditions are met.
+- **Missing Dependencies**: Ensure all dependent entities are included or exist in the target environment.
+- **API Rate Limits**: Be aware of Kontent.ai API rate limits when synchronizing large content models.
+
+---
+
+By following these guidelines and understanding the potential impacts, you can effectively use the `sync-model run` command to synchronize content models between environments, ensuring consistency and streamlining your content management workflows.
```

Comment:
this is also for me like the sentence which doesn't give me as a developer anything :D